src/components/FeatureViews/Scheduler/Scheduler.js

```diff
@@ -137,10 +139,11 @@ class Scheduler extends Component {
     }
 
     componentWillUnmount() {
-
+        console.log("test")
```

Comment:
delete

Reply:
I got the auth story in the spreadsheet, the Story/AC ID will be: L1Login/Authentication